ROMFS/px4fmu_common/init.d-posix/rcS

```diff
@@ -130,6 +130,7 @@ then
 	param set CAL_MAG_PRIME 197388
 
 	param set CBRK_AIRSPD_CHK 0
+	param set CBRK_SUPPLY_CHK 894281
```

Comment:
As a side effect of this we cannot test low battery reactions in SITL anymore. Instead, you get this sequence when setting the `SIM_BAT_MIN_PCT` to a low value and no reaction.
![image](https://user-images.githubusercontent.com/4668506/81569214-45e5e980-939f-11ea-8d86-bcca4050d3bb.png)

Reply:
Then something isn't quite right. SITL is actually the only "board" in upstream master that supports "shutdown". https://github.com/PX4/Firmware/blob/2bbdef205e163181cb8c02b08aeceb4c69101b36/boards/px4/sitl/src/board_shutdown.cpp#L52-L58

We might as well get rid of the fake shutdown.